Library/Homebrew/rubocops/cask/ast/stanza.rb

```diff
@@ -52,6 +52,16 @@ def same_group?(other)
           stanza_group == other.stanza_group
         end
 
+        def stanza_comments(stanza_node)
+          stanza_node.each_node.reduce([]) do |comments, node|
+            comments | comments_hash[node.loc]
+          end
+        end
+
+        def comments_hash
+          @comments_hash ||= Parser::Source::Comment.associate_locations(cask_node, stanza_comments(stanza_node))
```

Comment:
I wonder, does this work with something like `@method_node.parent` so we don't need to pass in `cask_node`?

Reply:
Also `stanza_node` should be `@method_node` here.

Reply:
It does. Good idea!